app/Services/PostService.php

```diff
@@ -43,8 +43,7 @@ public function create($data, $signupId, $transactionId)
 
         // Save the new post in Customer.io, via Blink.
         if (config('features.blink') && $should_send_to_blink) {
```

Comment:
Not needed for this PR, but it'd be nice to extract this check inside the job as well (so we always dispatch `SendPostToBlink`, but only make the API call inside the job if this feature flag is toggled on). That way the job always runs and logs the expected payload when we're running on local machines.